docs/Configuration.md

```diff
@@ -1,4 +1,4 @@
-By default, the mainnet bcoin config file will reside in ~/.bcoin/bcoin.conf.
+By default, the mainnet bcoin config files will reside in ~/.bcoin/bcoin.conf and ~/.bcoin/wallet.conf.
```

Comment:
I would like you to write here that testnet wallet.conf and regtest wallet.conf should be placed under `~/.bcoin/testnet` and `~/.bcoin/regtest` accordingly. I have stuck by this for hours.

Reply:
@joemphilips This would be a good addition, but it is not necessarily always the case: If you add `network: regtest` to the file at `~/.bcoin/bcoin.conf` bcoin will run in `regtest` mode even though the conf file is at the default mainnet location.

Reply:
updated